content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -34,57 +33,69 @@ microservices running under them.
+
+    {{< text bash >}}
+    $ kubectl delete authorization requestcontext -n istio-system
+    $ kubectl delete rbac handler -n istio-system
+    $ kubectl delete rule rbaccheck -n istio-system
+    {{< /text >}}
+
+    * Run the following commands to remove any existing RBAC policies:
+
+      > You could keep existing policies but you will need to make some changes to the `constraints` and `properties` field
+in the policy, see [constraints and properties](/docs/reference/config/authorization/constraints-and-properties/)
+for the list of supported keys in `constraints` and `properties`.
+
+    {{< text bash >}}
+    $ kubectl delete servicerole --all
+    $ kubectl delete servicerolebinding --all
+    {{< /text >}}
+
+* Point your browser at the Bookinfo `productpage` (http://$GATEWAY_URL/productpage). You should see:
 
-## Enabling Istio RBAC
+    * "Book Details" section in the lower left part of the page, including type, pages, publisher, etc.
+    * "Book Reviews" section in the lower right part of the page.
 
-Run the following command to enable Istio RBAC for "default" namespace.
+    If you refresh the page several times, you should see different versions of reviews shown in productpage,
+    presented in a round robin style (red stars, black stars, no stars)
 
-> If you are using a namespace other than `default`, edit the file `samples/bookinfo/platform/kube/istio-rbac-enable.yaml`,
-and specify the namespace, say `"your-namespace"`, in the `match` statement in `rule` spec
-`"match: destination.namespace == "your-namespace"`.
+## Enabling Istio authorization
+
+Run the following command to enable Istio authorization for "default" namespace:
 
 {{< text bash >}}
-$ istioctl create -f @samples/bookinfo/platform/kube/istio-rbac-enable.yaml@
+$ istioctl create -f @samples/bookinfo/platform/kube/rbac/rbac-config-ON.yaml@
 {{< /text >}}
 
 > If you have conflicting rules that you set in previous tasks, use `istioctl replace` instead of `istioctl create`.
 
-It also defines "requestcontext", which is an instance of the
-[authorization template](/docs/reference/config/policy-and-telemetry/templates/authorization/).
-"requestcontext" defines the input to the RBAC engine at runtime.
-
 Point your browser at the Bookinfo `productpage` (http://$GATEWAY_URL/productpage). Now you should see
-`"PERMISSION_DENIED:handler.rbac.istio-system:RBAC: permission denied."` This is because Istio RBAC is "deny by default",
-which means that you need to explicitly define access control policy to grant access to any service.
+`"RBAC: access denied"`. This is because Istio authorization is "deny by default", which means that you need to
+explicitly define access control policy to grant access to any service.
 
-> There may be delay due to caching on browser and Istio proxy.
+> There may be delay due to caching on browser and policy delayed delivered to Istio proxy.
```

Comment:
Replace with

There may be some delays due to caching and other propagation overhead.

Reply:
Done.